src/app/admin/backup.ts

```diff
@@ -22,13 +14,13 @@ export const uploadBackup =
   (logger: Logger) =>
   async ({ backup, pubkey }: { backup: string; pubkey: Pubkey }) => {
     logger.debug({ backup }, "updating scb on dbx")
-    const filename = `${BitcoinNetwork()}_lnd_scb_${pubkey}_${Date.now()}`
+    const filename = `${env.NETWORK}_lnd_scb_${pubkey}_${Date.now()}`
```

Comment:
Regardless of wether or not the value is being injected from the environment or yaml the application / service code should not know about this and continue getting the value from the config module.
There needs to be a translation layer between the config that the app layer consumes and the method of injection. Ideally this refactor should only affect the config module and not config consumers.

Or in other words - the only place where you are allowed to reference `env` directly is in `@config`

Reply:
I'm not sure there is a real benefits to that, beyond the boilerplate code?

there is 100 configs that we can/have to inject into the server, and it's already fully validated in env.ts

so the boilerplate just to do that https://github.com/GaloyMoney/galoy/pull/3008/files#diff-f0724f34ef85aed4dc757304299f2d7626984d7e04543f22241755c17a94d1beL90 feels unnecessary (leads to dead code) but also error prone (we had default settings in multiple place with different values)